setup.py

```diff
@@ -24,6 +24,7 @@
         "nose>=1.3.7",
         "parameterized>=0.5.0",
         "tox>=2.3.1",
+        "flake8==3.7.9",
```

Comment:
Should this be a lower bound, since this is the libraries requirements, as opposed to a lockfile?